editor/project_manager/project_dialog.cpp

```diff
@@ -844,6 +874,20 @@ void ProjectDialog::show_dialog(bool p_reset_name) {
 			default_files_container->hide();
 
 			callable_mp((Control *)project_path, &Control::grab_focus).call_deferred();
+		} else if (mode == MODE_DUPLICATE) {
+			set_title(TTR("Duplicate Project"));
+			set_ok_button_text(TTR("Duplicate"));
```

Comment:
```suggestion
			set_title(TTRC("Duplicate Project"));
			set_ok_button_text(TTRC("Duplicate"));
```

Reply:
Thanks! I've updated this (and all the `TTR()` to `TTRC()` suggestions) in my latest push